doc/release-notes-0.11.md

```diff
@@ -0,0 +1,12 @@
+This page discusses the changes that you need to be aware of when migrating your Ziti deployment from version 0.10.x to version 0.11.x
+
+# Theme
+Ziti 0.11 includes the following: 
+ 
+ * Ziti connections from Ziti SDK client to services hosted by SDK are encrypted end-to-end (no API changes)
```

Comment:
Rather than "are encrypted" should this say "can be encrypted"?

Reply:
"can be" sounds like it is an option left up to the user. with v0.11 SDK on both sides connection *will be* encrypted

Reply:
It should be optional, no?  E.g., the app is already end-to-end encrypted and the customer doesn't want/need the extra overhead.